database/migrations/default/1738088180607_notes_audit_fields/up.sql

```diff
@@ -0,0 +1,26 @@
+
+--
+-- update crash notes
+--
+alter table crash_notes add column created_by text, add updated_by text;
+
+update crash_notes set created_by = user_email, updated_by = user_email;
+
+alter table crash_notes
+alter column created_by set not null,
+alter column updated_by set not null,
+drop column user_email,
+drop column date;
```

Comment:
this `date` column is lingering from pre-data model days. we already have `created_at` and `updated_at` working on this tables